packages/controllers/src/snaps/SnapController.ts

```diff
@@ -1496,3 +1593,12 @@ function isValidSnapVersionRange(version: unknown): version is string {
     typeof version === 'string' && validRangeSemver(version) !== null,
   );
 }
+
+function snapIdToSnapPrefix(snapId: string): SnapIdPrefixes {
+  for (const prefix of Object.values(SnapIdPrefixes)) {
```

Comment:
Nit: Could be simplified to `const value = Object.values(SnapIdPrefixes).find(prefix => snapId.startsWith(prefix));`?